### PR TITLE
Create Hosted Engine disks with the corresponding content type

### PR DESCRIPTION
**Pull request by tnisan**

### CODE REVIEW

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for this project! 

I am the bot who triggers "standard-CI" builds for this project.
As a security measure, I will not run automated tests on PRs that are not from white-listed contributors.

In order to allow automated tests to run, please ask one of the project maintainers to review the code and then do one of the following:

1. Type `ci test please` on this PR to trigger automated tests for it.
2. Type `ci add to whitelist` on this PR to trigger automated tests for it and also add you to the contributor white-list so that your future PRs will be tested automatically. ( keep in mind this list might be overwritten if the job XML is refreshed, for permanent whitelisting, please follow #3 option )
3. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the project organization, so you'll be able to run tests for all the organization's projects.

**Comment by tnisan** [comment]
@tiraboschi please review

**Comment by irosenzw** [comment]
Looks OK to me, @tnisan did you verified it ?

**Comment by tnisan** [comment]
Not yet as I'm waiting for the API model, VDSM and the Engine patches to be merged first

File: library/ovirt_disk.py [hunk 1]
```diff
@@ -0,0 +1,768 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2016 Red Hat, Inc.
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
```
Comment (tiraboschi) [inline library/ovirt_disk.py]: Can you please add a comment here asking to remove it once we will require ansible 2.8?

**Comment by tiraboschi** [comment]
ci please build

File: tasks/create_target_vm/01_create_target_hosted_engine_vm.yml [hunk 2]
```diff
@@ -132,27 +133,31 @@
         size: "{{ he_disk_size_GB }}GiB",
         format: 'raw',
         sparse: "{{ false if he_domain_type == 'fc' or he_domain_type == 'iscsi' else true }}"
```
Comment (tiraboschi) [inline tasks/create_target_vm/01_create_target_hosted_engine_vm.yml]: , at the end

File: tasks/create_target_vm/01_create_target_hosted_engine_vm.yml [hunk 3]
```diff
@@ -132,27 +133,31 @@
...
+        content: 'hosted_engine'
       }
       - {
         name: 'he_sanlock',
         description: 'Hosted-Engine sanlock disk',
         size: '1GiB',
         format: 'raw',
         sparse: false
```
Comment (tiraboschi) [inline tasks/create_target_vm/01_create_target_hosted_engine_vm.yml]: same

File: tasks/create_target_vm/01_create_target_hosted_engine_vm.yml [hunk 4]
```diff
@@ -132,27 +133,31 @@
...
+        content: 'hosted_engine_sanlock'
       }
       - {
         name: 'HostedEngineConfigurationImage',
         description: 'Hosted-Engine configuration disk',
         size: '1GiB',
         format: 'raw',
         sparse: false
```
Comment (tiraboschi) [inline tasks/create_target_vm/01_create_target_hosted_engine_vm.yml]: same

File: tasks/create_target_vm/01_create_target_hosted_engine_vm.yml [hunk 5]
```diff
@@ -132,27 +133,31 @@
...
+        content: 'hosted_engine_configuration'
       }
       - {
         name: 'he_metadata',
         description: 'Hosted-Engine metadata disk',
         size: '1GiB',
         format: 'raw',
         sparse: false
```
Comment (tiraboschi) [inline tasks/create_target_vm/01_create_target_hosted_engine_vm.yml]: same

**Comment by tiraboschi** [comment]
ci please build

**Comment by tiraboschi** [comment]
ci please build

**Comment by tnisan** [comment]
This specific change is required in the Engine and was added to the 4.3.3.2 build

**Pull request merged**